File: A-Set/401 - 500/432 - Choosing Teams.cpp
#include <iostream>
#include <algorithm>
 
int main() {
  int numberOfMembers, times, size = 0, teamCtr = 0, ans = 0;
  int members[2000];
  
  std::cin >> numberOfMembers >> times;
  
  for(int i = 0; i < numberOfMembers; i++) {
    std::cin >> members[i];
    size++;
  }
  
  std::sort(members, members+size);
  
  for(int i = 0; i < size; i++) {
    if(members[i] + times <= 5) {
      teamCtr++;
    }
    
    if(teamCtr == 3) {
      ans++;
      teamCtr = 0;
    }
  }
  
  std::cout << ans;
  
  return 0;
}

File: A-Set/401 - 500/432 - Choosing Teams.py
teamCounter = 0
ans = 0
numberOfMembers, times = list(map(int, input().split()))
members = list(map(int, input().split()))
members.sort()

for i in members:
  if i + times <= 5:
    teamCounter += 1

  if teamCounter == 3:
    ans += 1
    teamCounter = 0

print(ans)